### ISSUE TITLE
Sofia not loading and other errors after 20200428T111313Z~30cf345b41

### ISSUE THREAD

**Issue by MarioG-X** [signalwire/freeswitch#604]
Sofia is not starting up, build looks fine, cannot access Sofia via command line.
This is weekly test using only downloaded source, no edited files. Make and startup log is attached.

Used git bisect to determine Sofia is not starting up and other errors below after April 27 changes by 30cf345b410dada4bb2f298d3eb1b577c8a4c59e. 
[make.txt](https://github.com/signalwire/freeswitch/files/4568606/make.txt)
[startup log.txt](https://github.com/signalwire/freeswitch/files/4568607/startup.log.txt)

Test using Apr 6 ae0444e9cbccdee55a80467d605e1e8c3363a36d worked.

Some unusual startup messages resulting from this error:2020-05-02 09:19:04.189786 [CRIT] 2020-05-02 09:18:32.284806 [ERR] switch_stun.c:900 STUN Failed! [Timeout]
2020-05-02 09:18:32.284843 [ERR] switch_xml.c:175 stun-set failed.
2020-05-02 09:18:37.287045 [ERR] switch_stun.c:900 STUN Failed! [Timeout]
2020-05-02 09:18:37.287097 [ERR] switch_xml.c:175 stun-set failed.

:1785 Error Loading module /usr/local/freeswitch/mod/mod_sofia.so
**Module load routine returned an error**

**Comment by andywolk** [comment]
Your stun configuration is failing.
Module load will fail if ext-rtp-ip or ext-sip-ip configuration param is set as empty string.
Because of this.
https://github.com/signalwire/freeswitch/commit/e753e615ca57675e5b291fd5f6c772fea1cfa528#diff-34a3fccf071006ee61693876874d0dcc

That was improved in latest master by adding this:
https://github.com/signalwire/freeswitch/pull/221

**Comment by andywolk** [comment]
Try adding different stun server

**Comment by MarioG-X** [comment]
The description states that this is the default configuration for weekly testing. There is no modification. It should do a clean make with all downloaded config/defaults. I just download, bootstrap, config, make, make install. So why would you need to add a STUN for that?

**Comment by andywolk** [comment]
https://github.com/signalwire/freeswitch/blob/master/conf/vanilla/vars.xml#L301
```
  <X-PRE-PROCESS cmd="stun-set" data="external_rtp_ip=stun:stun.freeswitch.org"/>
```
If this ip lookup fails there will be empty external_rtp_ip var and sofia won't load.
May explain why it works/does not work sometimes.

**Comment by MarioG-X** [comment]
That explains a lot, I think the stun server is a little wacky, never seen that since 2010.

I saw errors for stun.freeswitch.org yesterday so I pinged it and it worked so I know the lookup was good. From the tests below you'll see without me changing anything between different tests the problem appeared and disappeared randomly. For the tests from this morning, it even changed between a few minutes. It could be the server or the internet between here (North California) and the server, but remember, lookup always worked so I don't think it's internet related. So my hours of work doing git bisects would have not happened if I waited for stun.freeswitch.org to stabilize.

Yesterday:
About 4 hours later, I downloaded/built all defaults on one of the test partitions (10.15) again, and it WORKED. Previously it did not. I then tried a second system (10.14) and it WORKED. And then I tried the last system (10.13) and it FAILED with the same error!

Today IMPORTANT:
Test 1. This morning I tried 10.13 build/test from scratch again, the same as yesterday and Sofia WORKED! However there was an error after startup with signalwire connector and stun.freeswitch.org

Test 2. Then, about 45 minutes later while typing this, to collect log, I started it and although Sofia WORKED the startup log had these messages which were not in the first test, I only shut it down and restarted:
2020-05-03 13:37:10.626952 [ERR] switch_stun.c:900 STUN Failed! [Timeout]
2020-05-03 13:37:10.626993 [ERR] switch_xml.c:175 stun-set failed.

Test 3. I shutdown and restarted while typing this and it WORKED and those messages were gone.
